src/postgraphql/schema/exportPostGraphQLSchema.ts

```diff
@@ -0,0 +1,26 @@
+import { writeFileSync } from 'fs'
+import { graphql, GraphQLSchema } from 'graphql'
+import { introspectionQuery, printSchema } from 'graphql/utilities'
+import { graphql } from 'graphql'
+
+/**
+ * Exports a PostGraphQL schema by looking at a Postgres client.
+ */
+export default async function exportPostGraphQLSchema (
+  schema: GraphQLSchema,
+  options: {
+    exportPathJSON?: string,
+    exportPathSchema?: string,
+  } = {},
+): Promise<void> {
+  // JSON version
+  if (options.exportPathJSON) {
```

Comment:
`''` is not a valid file path though, so do we want to accept that? Then again, there's definitely a case to be made for handling `''` the same as other invalid paths by just throwing an error when the file write fails.